tests/unittests/test_options.py

```diff
@@ -10,7 +11,8 @@
 
 
 class MyOptions(Options):
-    config_section: str = "my_options"
+    config_section: ClassVar[str] = "my_options"
```

Comment:
Is it necessary to include the type annotation in every subclass?
Just curious, it won't hurt as you typically have other fields too that you need to annotate.

Reply:
It's not necessary, but when I tried first time, I got an error. Now it works ok, strange.